### ISSUE TITLE
Scroll to current time in calendar is annoying for desktop views

### ISSUE THREAD

**Issue by armhub** [tutao/tutanota#6522]
In a desktop view with a lot of space on the screen it is annoying when the calendar scrolls to the current time, especially when it is the the end of a day. In this case only the evening and night is visible but it would be much more important to see the full day, especially if the week view is visible and you also want to see the other day's events.

**Comment by marcosdiasalves** [comment]
This is true for mobile as well. Please implement a pinch-to-zoom feature so we can see the entire day at once.

**Comment by armhub** [comment]
This is already fixed.
